server/views/components/image-gallery/index.njk

```diff
@@ -1,12 +1,19 @@
 <div class="image-gallery">
-{% for item in imageGallery.items %}
-  <div class="image-gallery__item">
-    {% component 'figure', {
-      type: 'picture',
-      sources: [{src: item.contentUrl, size: item.width}],
-      caption: item.caption,
-      icon: 'other/picture'
-    } %}
-  </div>
-{% endfor %}
+  {% for item in imageGallery.items %}
+    <div class="image-gallery__item">
+      <figure class="figure image-gallery__figure">
+        <picture class="figure__picture">
```

Comment:
Correct, and good spot.
It's in the current [figure](https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/views/components/figure/index.njk) component.

I'll leave it here for consistency, but will be looking at refactoring the figure component soon anyway...